cli/cli/kurtosis_gateway/connection/connection.go

```diff
@@ -147,7 +182,8 @@ func newLocalPortToPodPortConnection(kubernetesRestConfig *k8s_rest.Config, podP
 }
 
 func (connection *gatewayConnectionToKurtosisImpl) Stop() {
-	connection.portforwarder.Close()
+	logrus.Infof("Closing connection to pod: %s", connection.urlString)
```

Comment:
@laurentluce this may be nitpicking a bit, but maybe we should generalize the language: 
"Closing connection to pod" => "Closing connection to service"
Our main use of the gateway is for k8s, but we could potentially apply this to other backends.